README.md

```diff
@@ -17,4 +17,6 @@ Please clone and star this repo to stay up to date on changes.
 
 **004 - Create Dynamic Serverless Loadbalancers** [[ Source ](https://github.com/premisedata/gcp-tutorials/004-dynamic-serverless-loadbalancer)] [[Written](https://austennovis.medium.com/e15751853312)] - A simple, scalable, and customizable solution to deploy and maintain  load balancers for any number of serverless applications.
 
-**005 - Bucket Replication** [[ Source ](https://github.com/premisedata/gcp-tutorials/005-bucket-replication)] [Written]()] - A simple way to set up bucket replication, leveraging goblet's use of decorators to set up cloud function triggers.
\ No newline at end of file
+**005 - Bucket Replication** [[ Source ](https://github.com/premisedata/gcp-tutorials/005-bucket-replication)] [Written]()] - A simple way to set up bucket replication, leveraging goblet's use of decorators to set up cloud function triggers.
+
+**006 - Service Account Key Rotater** [[ Source ](https://github.com/premisedata/gcp-tutorials/006-service-account-key-rotater)] [Written]()] - A seamless way to set up Service Account Key rotation within Github Secrets and Cloud Storage
```

Comment:
instead of seemless lets use 
``` A pluginable solution to automate Service Account Key rotations that currently supports Github Secrets and Cloud Storage.```